glean-core/pings.yaml

```
@@ -29,14 +29,14 @@ baseline:
   reasons:
     dirty_startup: |
       The ping was submitted at startup, because the application process was
-      killed before the Glean SDK had the chance to generate this ping, when
-      going to background, in the last session.
+      killed before the Glean SDK had the chance to generate this ping, before
+      becoming inactive, in the last session.
 
       *Note*: this ping will not contain the `glean.baseline.duration` metric.
-    background: |
-      The ping was submitted before going to background.
-    foreground: |
-      The ping was submitted when the application went to foreground, which
+    inactive: |
+      The ping was submitted before becoming inactive.
```

While "before" is technically correct, what do you think about making it more explicit that this is triggered by becoming inactive? 

```suggestion
      The ping was submitted when becoming inactive.
```